src/vm/unixlowmem.cpp

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ===========================================================================
+
+#include "common.h"
+#include "unixlowmem.h"
+#include <stdlib.h>
+#include "debugmacros.h"
+
+#ifdef FEATURE_PAL
+
+UnixLowMemoryDetector::UnixLowMemoryDetector()
+{
+    LIMITED_METHOD_CONTRACT;
+
+    auto percent = max(min(ReadLowMemoryLimitPercent(), 100), 1);
```

Comment:
Can you please use specific type instead of auto? We use auto mostly just for lambdas or iterators in the runtime.